schema.sql

```diff
@@ -0,0 +1,68 @@
+CREATE DATABASE CatalogOfThings;
+USE CatalogOfMyThings;
+
+CREATE TABLE item (
+    id INT PRIMARY KEY,
+    title VARCHAR(255),
+    publish_date DATE,
+    publisher VARCHAR(255),
+    cover_state VARCHAR(255)
+    archived BOOLEAN,
+    label_id INT, -- Add a foreign key for the label (one-to-many relationship)
+    genre_id INT, -- Add a foreign key for the genre (one-to-many relationship)
+    author_id INT, -- Add a foreign key for the author (one-to-many relationship)
+    FOREIGN KEY (label_id) REFERENCES labels (id)  
+    FOREIGN KEY (genre_id) REFERENCES genres (id),
+    FOREIGN KEY (author_id) REFERENCES authors (id),
+);
+
+--create music album table--
+CREATE TABLE MUSIC_ALBUM(
+    ID INTEGER PRIMARY KEY,
+    ON_SPOTIFY BOOLEAN,
+    ARCHIVED BOOLEAN,
+    PUBLISH_DATE DATE,
```

Comment:
- [x] the `music_album` table should have only 2 columns `on_spotify` and `id`(which is referencing the item id). Please fix that to match the given [UML diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png)

![image](https://github.com/Uzair-Manzoor/RubyCapstone/assets/68796232/3eaa1487-121b-471e-9888-4d7941512265)

```suggestion
CREATE TABLE music_album(
	id INT NOT NULL,
	on_spotify BOOLEAN,
	FOREIGN KEY (id) REFERENCES item (id)
);
```